components/RuiBreadcrumbs/src/RuiBreadcrumbs.ts

```diff
@@ -0,0 +1,64 @@
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import { LitElement, html, property, CSSResultArray, TemplateResult} from 'lit-element';
+import { variables, layout } from './RuiBreadcrumbs.css'
+
+
+export class RuiBreadcrumbs extends LitElement {
+	/**
+	 * The location array of the breadcrumbs
+	 */
+
+	@property({type : String})
+	public location?: string = '';
+
+
+
+	/**
+    *
+    * The styles for breadcrumbs
+    * @remarks
+    * If you are extending this class you can extend the base styles with super. Eg `return [super(), myCustomStyles]`
+    */
+    public static get styles(): CSSResultArray {
+        return [variables, layout];
+    }
+
+    /* #endregion */
+
+
+    /* #region Methods */
+
+    /**
+    * Render method
+        * @slot This is a slot test
+    */
+
+    public render(): TemplateResult {
+		if (this.location) {
+			const location = JSON.parse(this.location);
+			const lastLocation = location.pop();
+
+			return html`
+				<div class="breadcrumb">
```

Comment:
Can you please refer to the W3C design pattern so we have a semantically correct markup?

https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190207/examples/breadcrumb/index.html

Reply:
Sure, Have changed this to the suggested template in w3